dev/VSIX/ItemTemplates/Desktop/CppWinRT/BlankWindow/BlankWindow.h

```diff
@@ -10,6 +10,7 @@ namespace winrt::$rootnamespace$::implementation
         {
             // Xaml objects should not call InitializeComponent during construction.
             // See https://github.com/microsoft/cppwinrt/tree/master/nuget#initializecomponent
+            ExtendsContentIntoTitleBar(true); // provides default WinUI custom title bar experience
```

Comment:
>ExtendsContentIntoTitleBar(true); // provides default WinUI custom title bar experience [](http://example.com/codeflow?start=12&length=87)

This feels incomplete. Should the template include a full example (similar to what's shown in [https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.window.settitlebar?view=windows-app-sdk-1.4#microsoft-ui-xaml-window-settitlebar(microsoft-ui-xaml-uielement](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.window.settitlebar?view=windows-app-sdk-1.4#microsoft-ui-xaml-window-settitlebar(microsoft-ui-xaml-uielement))) to demonstrate the complete usage?

Reply:
this is the complete use.  Calling this api creates a default custom titlebar which extends the entire width of the window i.e. same area as system titlebar would have taken.  the docs are not updated strangely. I have had updated them while back.  See my merged PR : Merged docs PR : https://github.com/MicrosoftDocs/winapps-winrt-api/pull/89
 I will dig that up. Thanks for bringing this to my attention.